recursive/recursive_sorting.js

```diff
@@ -49,9 +83,55 @@ const quick_sort = arr => {}
  */
 
 //helper function to merge 2 sorted arrays
-const merge = (arrA, arrB) =>{}
+const merge = (arrA, arrB) =>{
+    let new_arr = []
+
+    while(arrA.length && arrB.length){
+        if(arrA[0] >= arrB[0]){
+            new_arr.push(arrB.shift())
+        }
+        else{
+            new_arr.push(arrA.shift())
+        }
+    }
+
+    while(arrA.length){
+        new_arr.push(arrA.shift())
+    }
+
+    while(arrB.length){
+        new_arr.push(arrB.shift())
+    }
+
+    return new_arr
+}
+
+const merge_sort = arr => {
+    let split_arr = []
```

Comment:
Another way to split the array is 
`const center = arr.length / 2;
    let left = arr.slice(0, center);
    let right = arr.slice(center);`